packages/babel-preset-alloy/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@alloy-js/babel-preset",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
why is this a patch bump while the other packages get a minor bump?

Reply:
babel preset is on a different ruleset because it's rarely changing.